server/channels/app/export.go

```diff
@@ -150,6 +151,15 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		updateJobProgress(ctx.Logger(), a.Srv().Store(), job, "attachments_exported", len(attachments)+len(directAttachments)+len(emojiPaths))
 	}
 
+	if opts.IncludeProfilePictures {
+		for _, profilePicture := range profilePictures {
+			if err := a.exportFile(outPath, profilePicture, zipWr); err != nil {
```

Comment:
If I understand this correctly, if there is an error during the export process, it will return the error and stop the execution. If that is the case, wouldn't it be better to gracefully handle the error and have a default profile image exported?

Reply:
Yeah, I agree with Pablo

Reply:
I think it would be better if I log something and then do nothing. Generating a default one means risking a new error that would also be unmanaged. The new mattermost server (import side) will generate a default one for the user if they don't have any. Does that seem acceptable? @pvev @BenCookie95

Reply:
Yeah sounds good to me @JulienTant

Reply:
I have changed it. Note that any other type of file exported (emojis, attachments) will prevent the export from completing, so this becomes a bit of an exception.